specreduce/tests/test_extinction.py

```diff
@@ -58,12 +61,17 @@ def test_missing_extinction_unit():
     """
     wave = np.linspace(0.3, 2.0, 50)
     extinction = 1. / wave
-    ext = AtmosphericExtinction(extinction=extinction, spectral_axis=wave * u.um)
-    assert(len(ext.extinction_mag) > 0)
-    assert(len(ext.transmission) > 0)
+    with pytest.warns(AstropyUserWarning):
+        ext = AtmosphericExtinction(extinction=extinction, spectral_axis=wave * u.um)
+        assert(len(ext.extinction_mag) > 0)
+        assert(len(ext.transmission) > 0)
 
 
 def test_transmission_model():
-    ext = AtmosphericTransmission()
-    assert(len(ext.extinction_mag) > 0)
-    assert(len(ext.transmission) > 0)
+    """
+    Test creating of default atmospheric transmission model
+    """
+    with pytest.warns(RuntimeWarning):
+        ext = AtmosphericTransmission()
+        assert(len(ext.extinction_mag) > 0)
+        assert(len(ext.transmission) > 0)
```

Comment:
```suggestion
    assert len(ext.extinction_mag) > 0
    assert len(ext.transmission) > 0
```

Reply:
thanks for the reminder to fix the `assert` syntax. that was bugging me, but hadn't gotten around to dealing with it. fixed now.